client/src/Atoms/NavLinkStyled/NavLinkStyled.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+const bgHandler = (props) => {
+  if (props.stylefor === 'Sidebar')
+    return `background: ${props.theme.primary101}`;
+  if (props.stylefor === 'Cornermenu')
+    return `color: ${props.theme.grey400}; font-weight: normal`;
+  if (props.stylefor === 'Navbar') return `color: ${props.theme.primary801}`;
```

Comment:
Perhaps you could add a default case? if none of the above -> then render stg

Reply:
If non of te above, css style reverts to what is defined outside the handler function. That is for the stylefor='Sidebar": no left padding and no background. I will clarify if this is 100% ok in the Navbar PR - which is also to be reviewed. See you there!